.github/workflows/build-package.yml

```diff
@@ -36,14 +37,21 @@ jobs:
         TARGET_PLATFORM:
           - centos,8
           - centos,7
-          - debian,bullseye
-          - debian,buster
-          - debian,stretch
           - oraclelinux,8
           - oraclelinux,7
-          - ubuntu,focal
-          - ubuntu,bionic
-          - pgxn
+        POSTGRES_VERSION:
+          - 10
+          - 11
+          - 12
+          - 13
+          - 14
```

Comment:
Shouldn't the PG values be inferred from the matrix?

Reply:
Actually no. We need to create each of the supported packaging images. Matrix is for to choose the postgres version releated to citus version. For example we may release 8.x 10.x and 11.x at the same time. For centos-8, we need pg 10 and 11 for 8.x , 12,13 and 14 for 10.x and 13 and 14 for 11.x. So we need all the images at the same time.